internal/server/metrics.go

```diff
@@ -35,7 +35,7 @@ func setupMetrics(db *data.DB) *prometheus.Registry {
 		Name:      "groups",
 		Help:      "The total number of groups",
 	}, []string{}, func() []metrics.Metric {
-		count, err := data.GlobalCount[models.Group](db)
+		count, err := data.QuickGlobalCount[models.Group](db)
```

Comment:
Why do some of these use quick and others the old GlobalCount ?

Reply:
You can't do a quick count on a global count that has where conditions

Reply:
Cool, makes sense. I'll be looking to remove `GlobalCount` when we replace gorm, and change it to something like:

```go
type CountOptions struct {
    // AllOrganizations instructs count to return a global count, not only the
    // count for the request organization.
    AllOrganizations true
}

func CountGrants(tx ReadTxn, opts CountOptions) (int64, error) { ... }

```

Reply:
that's fine. Global counts are generally a pain. A bunch of the counts we're doing aren't going to scale, and that includes the ones currently in pagination.